gax/operation.h

```diff
@@ -0,0 +1,118 @@
+// Copyright 2019 Google Inc.  All rights reserved
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef GAPIC_GENERATOR_CPP_GAX_OPERATION_H_
+#define GAPIC_GENERATOR_CPP_GAX_OPERATION_H_
+
+#include "call_context.h"
+#include "internal/gtest_prod.h"
+#include "operations_stub.h"
+#include "status.h"
+#include "status_or.h"
+
+#include "google/longrunning/operations.pb.h"
+
+#include <iostream>
+
```

Comment:
nit: why this blank line here?

Reply:
clang-format is aggressive about reordering include paths into (I think) alphabetical order. I like to have system includes after everything else so that it's a little easier to not rely on implicit includes that go away unexpectedly. Adding a newline between local and system includes prevents clang-format from rearranging the includes.